qemu/tests/cfg/block_hotplug.cfg

```diff
@@ -34,10 +42,16 @@
         - with_repetition:
             repeat_times = 300
         - with_reboot:
+            sub_type_after_plug = boot
```

Comment:
After you add this param, script will execute twice boot before unplug device.

Reply:
@aliang123  add this param can make sure the guest will reboot after hotplug a disk, will only cause one time reboot, after guest booted, will unplug the disk and do guest reboot~thus, it can meet our test requirement:
Reboot guest after hotpulg a disk, and Reboot guest after hotunplug a disk.

Thanks

Reply:
sorry for mistake, just reboot after unplug.